easybuild/easyconfigs/m/MicroHH/MicroHH-2.0.1-foss-2025a.eb

```
@@ -0,0 +1,76 @@
+# Author: Bert Jorissen (Universiteit Antwerpen)
+
+easyblock = 'CMakeMake'
+
+name = 'MicroHH'
+version = '2.0.1'
+
+homepage = 'https://microhh.org/'
+description = """
+MicroHH is a computational fluid dynamics code designed to simulate turbulent flows in the atmosphere using 
+Direct Numerical Simulation (DNS) and Large-Eddy Simulation (LES) techniques.
+Its capabilities span a range from highly idealized flows to realistic atmospheric boundary layers,
+incorporating all relevant processes, such as moist thermodynamics, radiation, land surface processes, and microphysics.
+The code is written in C++/CUDA and runs on both CPUs and GPUs.
+"""
+
+toolchain = {'name': 'foss', 'version': '2025a'}
+
+sources = [{
+    'filename': '%(version)s.tar.gz',
```

```suggestion
    'filename': SOURCE_TAR_XZ,
```
please use this template, we need xz for reproducible checksums of git checkouts